### DISCUSSION TITLE
milimeters instead of FEET

### DISCUSSION THREAD

**Discussion by RichardPinka** [jeremytammik/RevitLookup#270] [Ideas]
Hello, after evaluating functionality i experience small unimportant detail. 
Duct element showed in my (metric based) model
shows its widht and height in FEET . 
![image](https://github.com/user-attachments/assets/14d356b3-e18e-4715-b46b-112966a1e1ef)
It would be nice, to have ability to set units in metric system or imperial (US) system.

**Comment by jeremytammik** [comment]
That does not make much sense to me. RevitLookup is a database exploration tool. [All Revit database length units are in imperial feet](https://thebuildingcoder.typepad.com/blog/2011/03/internal-imperial-units.html). That cannot be changed. Only the Revit UI can be set up to display more convenient user-oriented units. I think RevitLookup better stick to the pure database data to avoid confusion. Maybe you are confused yourself, making such a request?

**Reply by RichardPinka** [reply]
Hello Jeremy, of course I was confused :D   Do not consider it like serious request. I understood now, that data is particular units, and this is not important to solve within the plugin. 
I will evaulate more how to dig into the ducts and piping structures and load the piping and ducts data from it by Revit API - Python

thanks for reaciton

**Comment by jeremytammik** [comment]
my pleasure, and thank you for asking and for your appreciation. ok, good luck diving in deeper. i'll close this for now then. cheers.

**Comment by jeremytammik** [comment]
re-discussed and confirmed in #305
